app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="photo_buttons">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:background">@drawable/circle</item>
+        <item name="android:scaleType">center</item>
+        <item name="android:tint">@color/photo_buttons</item>
```

Comment:
This is a small thing but can you rename this photo_buttons because a style and a color both are declared by the same name!